src/components/Wrapper.js

```diff
@@ -16,15 +15,14 @@ import {
 } from 'ramda';
 import { DesktopOnly, MobileOnly } from './MediaQuery';
 import MenuButtons from './MenuButtons';
-import Spacer from './Spacer';
 
 const drawerWidth = 142;
```

Comment:
We can remove this variable along with any references to it (lines 32-34 and just get rid of the `Main` component I guess, and just use the standard `main` tags in the markup)

Reply:
I was really confused because I thought I did this already, then realized it was in my new branch for prospects haha. I've cleaned it up more in here, and got rid of the 'Main' styled component.